examples/knative/python/kn-py-vm-attr/function.yaml

```diff
@@ -0,0 +1,94 @@
+---
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: kn-py-vm-attr-service
+  labels:
+    app: kn-py-vm-attr
```

Comment:
I'll keep this note here for reference as I'm not sure if the newly added VEBA UI will have issues when this label != service name. Let's keep this comment open here in case we're running into issues.